tests/packaging/test_rest_api.sh

```diff
@@ -7,31 +7,42 @@ unset PYTHONPATH
 
 source "${virtualenv_activate}"
 
+# Start Girder server
 export GIRDER_PORT=31200
 python -m girder &> /dev/null &
 
+# Ensure the server started
+girder_pid=$!
+sleep 1
+if ! ps -p $girder_pid &> /dev/null ; then
+    echo "Error: Girder could not be started"
+    exit 1
+fi
+
 version="$(girder-install version)"
 echo "Detected api version ${version}"
 
-TIMEOUT=0
-until [ $TIMEOUT -eq 15 ]; do
-    json=$("${CURL}" -s http://localhost:${GIRDER_PORT}/api/v1/system/version)
+timeout=0
+until [ $timeout -eq 5 ]; do
+    json=$("${CURL}" --max-time 5 --silent http://localhost:${GIRDER_PORT}/api/v1/system/version)
```

Comment:
With curl, I'd recommend also specifying `--connect-timeout 5` (besides the `--max-time` parameter).  In completely unrelated use of curl, I've had it stuck in the connection phase even with the max-time specified.  As long as we are adding the timeouts, we should have both of them.